mtz-ajax-interceptor-behavior.html

```diff
@@ -1,13 +1,136 @@
 <link rel="import" href="../polymer/polymer.html">
 
-<!--
-`mtz.AjaxInterceptorBehavior`
+<script>
+  window.mtz = window.mtz || {};
 
-@demo demo/index.html
--->
+  /**
+   * `mtz.AjaxInterceptorBehavior` handles registering declarative interceptors then listening for all registered events
+   *  and calling the interceptors associated with those events.
+   *
+   * @polymerBehavior mtz.AjaxInterceptorBehavior
+   */
+  mtz.AjaxInterceptorBehavior = {
+    properties: {
+      /* The map of event to interceptor Set to listen against */
```

Comment:
The wording of this comment is a bit confusing. Revise to be more clear. Maybe just "Event to interceptor map"?